src/views/Settings/GoTVPreferences.tsx

```diff
@@ -48,33 +64,116 @@ export function GoTVPreferences(): JSX.Element {
+                    <Select
+                        options={languageOptions}
+                        placeholder={_("All Languages")}
+                        isMulti
+                        value={languageOptions.filter((option) =>
+                            selectedLanguages.includes(option.value),
+                        )}
+                        onChange={handleLanguageChange}
+                        className="language-select"
+                        classNamePrefix="ogs-react-select"
+                    />
+                </PreferenceLine>
+
+                <PreferenceLine title={_("Show active streams indicator in navbar")}>
+                    <Toggle checked={showGoTVIndicator} onChange={toggleGoTVIndicator} />
+                </PreferenceLine>
+
+                <PreferenceLine title={_("Automatically play top stream on load")}>
+                    <Toggle checked={autoSelect} onChange={toggleAutoSelect} />
+                </PreferenceLine>
+
+                <PreferenceLine title={_("Show mature streams")}>
+                    <Toggle checked={allowMatureStreams} onChange={toggleAllowMatureStreams} />
+                </PreferenceLine>
+
+                <PreferenceLine title={_("Allow Notifications")}>
+                    <Toggle checked={allowNotifications} onChange={setAllowNotifications} />
+                </PreferenceLine>
+
+                <PreferenceLine
+                    title={_("Connect your Twitch account")}
+                    description="Link your Twitch account to get notifications when your favorite streamers are actively streaming Go"
+                >
+                    {isAuthenticated ? (
+                        <button disabled>{_("Connected")}</button>
+                    ) : (
+                        <button onClick={authenticateWithTwitch}>
+                            {_("Authenticate with Twitch")}
+                        </button>
                     )}
-                    onChange={handleLanguageChange}
-                    className="language-select"
-                    classNamePrefix="ogs-react-select"
-                />
-            </PreferenceLine>
-
-            <PreferenceLine title={_("Show active streams indicator in navbar")}>
-                <Toggle checked={showGoTVIndicator} onChange={toggleGoTVIndicator} />
-            </PreferenceLine>
-
-            <PreferenceLine title={_("Automatically play top stream on load")}>
-                <Toggle checked={autoSelect} onChange={toggleAutoSelect} />
-            </PreferenceLine>
-
-            <PreferenceLine title={_("Show mature streams")}>
-                <Toggle checked={allowMatureStreams} onChange={toggleAllowMatureStreams} />
-            </PreferenceLine>
+                    {isTokenExpired && <p>{_("Token expired. Please re-authenticate.")}</p>}
+                </PreferenceLine>
+            </div>
+
+            {/* TODO: Remove before deployment */}
```

Comment:
If you think this might still be useful, you could hide it behind a `user.is_moderator` check or something